cpp/examples/deck.gl/flight-paths.cc

```diff
@@ -47,18 +47,13 @@ auto createLineLayer(const std::string &dataPath) -> std::shared_ptr<LineLayer::
   lineLayerProps->id = "flight-paths";
   lineLayerProps->opacity = 0.8f;
   lineLayerProps->getSourcePosition = [](const Row &row) -> mathgl::Vector3<float> {
-    // NOTE: Decimal array data is being loaded as double
-    auto startPositions = row.getDoubleVector3("start");
-    return mathgl::Vector3<float>{startPositions};
+    return row.getVector3<float>("start");
```

Comment:
If I understand correctly, the method signature would end up being `auto get(const std::string& columnName, const T& defaultValue = {}) const -> T`?

That doesn't sound like a good idea to me as whoever is calling this might expect us to support any type out there, while in reality it would be only vectors. We could throw an error for when specialization isn't implemented for that type, but it just seems unnecessary  and it does not seem to add too much value